csharp/src/Drivers/Databricks/DatabricksConnection.cs

```diff
@@ -574,5 +581,14 @@ protected override void ValidateOAuthParameters()
                 base.ValidateOAuthParameters();
             }
         }
+
+        internal static String? HandleSparkCatalog(String? CatalogName)
```

Comment:
```suggestion
        internal static string? HandleSparkCatalog(string? catalogName)
```
That is, use the type alias `string` instead of the type name `String` and use a lower-case argument name.

Reply:
ah, good catch thanks!